cli/cli_app.go

```diff
@@ -404,6 +409,21 @@ func runTerragruntWithConfig(terragruntOptions *options.TerragruntOptions, terra
 	return errors.NewMultiError(beforeHookErrors, terraformError, postHookErrors)
 }
 
+// The Terragrunt configuration can contain a set of inputs to pass to Terraform as environment variables. This method
+// sets these environment variables in the given terragruntOptions.
+func setTerragruntInputsAsEnvVars(terragruntOptions *options.TerragruntOptions, terragruntConfig *config.TerragruntConfig) error {
+	asEnvVars, err := toTerraformEnvVars(terragruntConfig.Inputs)
+	if err != nil {
+		return err
+	}
+
+	for key, value := range asEnvVars {
+		terragruntOptions.Env[key] = value
+	}
```

Comment:
So this means we can't set the env var `TF_VAR_xxx` outside of terragrunt to override the inputs?

Reply:
Thanks for the great catch. Fixed and added tests:

https://github.com/gruntwork-io/terragrunt/pull/731/commits/fdce37cf37b5f8de94eee07d9e8b097cf45e92fa
https://github.com/gruntwork-io/terragrunt/pull/731/commits/993e1957a1f23e11961444b88f9ac94a47176a80